src/modules/charging-import/lib/queries/charging.js

```diff
@@ -132,7 +132,71 @@ on conflict (external_id) do update set
   date_updated=NOW();
 `;
 
+const importChargeVersions = `INSERT INTO water.charge_versions (licence_ref, scheme, external_id, version_number, start_date, status, apportionment,
+error, end_date, billed_upto_date, region_code, date_created, date_updated, source, invoice_account_id, company_id, licence_id, change_reason_id)
+SELECT
+  l."LIC_NO" AS licence_ref,
+  'alcs' AS scheme,
+  concat_ws(':', v."FGAC_REGION_CODE", v."AABL_ID", v."VERS_NO") as external_id,
+  v."VERS_NO"::integer AS version_number,
+  cvm.start_date AS start_date,
+  cvm.status::water.charge_version_status,
+CASE v."APPORTIONMENT"
+  WHEN 'Y' THEN true
+  WHEN 'N' THEN false
+END AS apportionment,
+CASE v."IN_ERROR_STATUS"
+  WHEN 'Y' THEN true
+  WHEN 'N' THEN false
+END AS error,
+cvm.end_date AS end_date,
+CASE v."BILLED_UPTO_DATE"
+  WHEN 'null' then null
+  ELSE to_date(v."BILLED_UPTO_DATE", 'DD/MM/YYYY')
+END AS billed_upto_date,
+v."FGAC_REGION_CODE"::integer AS region,
```

Comment:
split_part(cvm.external_id, ':', 1) as region

Reply:
Good shout. 

Went for split_part(cvm.external_id, ':', 1)::integer as region, because it was upset about it being a string rather than int.